### PR TITLE
Add welcome bot

### PR DESCRIPTION
**Pull request by rullzer**
I installed the welcome bot.

This adds the message for new contributors that have their first PR merged.

We can also add:
 * First issue
 * Fist PR opened

See: https://github.com/behaviorbot/welcome?installation_id=457148&setup_action=install

@jospoortvliet @jancborchardt feel free to adjust the wording.

### CODE REVIEW

**Comment by skjnldsv** [comment]
Could be nice to also have links to the irc freenode? 
Maybe also our code of conduct? I dunno :)

**Comment by rullzer** [comment]
@skjnldsv good point! But maybe we could do that also when the first PR is opend ;)

anyway that is why I wanted input. As I'm sure we also probably need fancy emojis etc ;)

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+firstPRMergeComment: >
+    Thanks for your first pull request and welcome to the club!
```
Comment (MorrisJobke) [inline .github/config.yml]:
```suggestion
    Thanks for your first pull request and welcome to the community!
```

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    Feel free to keep them comming!
```
Comment (MorrisJobke) [inline .github/config.yml]:
```suggestion
    Feel free to keep them coming! If you are looking for issues to tackle then have a look at this selection: https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22
```

**Review by MorrisJobke — CHANGES_REQUESTED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**